README.md

```diff
@@ -98,25 +74,22 @@ Full example:
 
 [Check out all the animations here!](https://daneden.github.io/animate.css/)
 
-## Usage
-
-To use animate.css in your website, simply drop the stylesheet into your document's `<head>`, and add the class `animated` to an element, along with any of the animation names. That's it! You've got a CSS animated element. Super!
+It's possible to change the duration of your animations, add a delay or change the number of times that it plays:
 
-```html
-<head>
-  <link rel="stylesheet" href="animate.min.css">
-</head>
+```css
+.yourElement {
+  animation-duration: 3s;
+  animation-delay: 2s;
+  animation-iteration-count: infinite;
+}
 ```
 
-or use the version hosted by [CDNJS](https://cdnjs.com/libraries/animate.css)
+_Note: be sure to replace "vendor" in the CSS with the applicable vendor prefixes (webkit, moz, etc)._
 
-```html
-<head>
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css">
-</head>
-```
 
-You can do a whole bunch of other stuff with animate.css when you combine it with jQuery or add your own CSS rules. Dynamically add animations using jQuery with ease:
+## Usage with jQuery
+
+You can do a whole bunch of other stuff with animate.css when you combine it with jQuery. A simple example:
```

Comment:
We should probably replace this with vanilla JS examples soon, but I'm ok with punting on that until a later date.

Reply:
You are completely right. I'll do it as soon as I get a bit of free time.

Reply:
As it's better to merge this PR asap, let's move it on #875, alright?

Reply:
Perfect!